web/frontend/src/language/de.json

```diff
@@ -65,6 +65,11 @@
     "notLoggedInActionText3": " um diese Aktionen durchzuführen.",
     "loginCallback": "Wir fahren mit der Authentifizierung fort. Sie sollten  weitergeleitet werden...",
     "logout": "Abmeldung",
+    "changeId": "UserId ändern",
```

Comment:
please change this to SCIPER id as well, since it can be pretty confusing to change the terminology